marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -695,4 +693,10 @@ private void updateProductContentForNonStandardProduct(List<GHContent> ghContent
       productModuleContentRepo.save(initialContent);
     }
   }
+
+  private void updateProductMarketplaceData(Product product) {
```

Comment:
We just need product.id for the query, passing a whole entity is hard to maintain & detect which method can modify it before save later.

Reply:
Moreover, the method name does not reflect its execution ( we just save the current instance once again if it exist & initialize if not)